lib/js/atoms/Tile/Tile.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+	<ripple-wrapper :disable="!interactive">
+		<div :class="{ '-interactive': interactive }" class="a-tile">
+			<icon
+				v-if="leftIcon"
+				:icon="leftIcon"
+				:size="ICON_SIZES.SMALL"
+				class="a-tile__leftIcon"
+			/>
+			<div class="a-tile__center">
+				<span class="a-tile__eyebrowText" v-text="eyebrowText" />
+				<span class="a-tile__text" v-text="text" />
+			</div>
+			<icon
+				v-if="interactive"
+				:icon="ICONS.FA_CHEVRON_RIGHT"
+				:size="ICON_SIZES.SMALL"
+				class="a-tile__interactiveIcon"
+			/>
+			<icon
+				v-else-if="rightIcon"
+				:icon="rightIcon"
+				:size="ICON_SIZES.SMALL"
+				class="a-tile__rightIcon"
+			/>
+			<div v-else-if="additionalText" class="a-tile__additionalText">{{
+				additionalText
+			}}</div>
+		</div>
+	</ripple-wrapper>
+</template>
+
+<style lang="scss" scoped>
+@import '../../../styles/settings/typography';
+@import '../../../styles/settings/spacings';
+@import '../../../styles/settings/colors/tokens';
+
+::v-deep .ripple {
+	background-color: $color-neutral-ripple !important;
+}
+
+.a-tile {
+	width: 100%;
+	min-height: $space-xl;
```

Comment:
`min-height` is suspected. Usually, height of the element is computed from padding and what's inside